internal/status/gatewayclass.go

```diff
@@ -14,12 +14,29 @@
 package status
 
 import (
+	"k8s.io/apimachinery/pkg/util/sets"
 	gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+	"github.com/envoyproxy/gateway/test/conformance"
 )
 
+// GetSupportedFeatures returns a list of supported Gateway-API features,
+// based on the running conformance tests suite.
+func GetSupportedFeatures() []gwapiv1.SupportedFeature {
+	supportedFeatures := conformance.EnvoyGatewaySuite.SupportedFeatures
+	supportedFeatures.Delete(conformance.EnvoyGatewaySuite.ExemptFeatures.UnsortedList()...)
+
+	ret := sets.New[gwapiv1.SupportedFeature]()
+	for _, feature := range supportedFeatures.UnsortedList() {
+		ret.Insert(gwapiv1.SupportedFeature(feature))
+	}
+	return sets.List(ret)
+}
```

Comment:
consider computing this only once. otherwise you would do this computation every time the GWC is reconciled. 
Similar to the feedback I got in https://github.com/istio/istio/pull/48770

Reply:
Good idea, thanks!